hieradata_aws/staging.yaml

```diff
@@ -159,6 +159,8 @@ govuk::apps::licencefinder::override_search_location: 'https://search-api.stagin
 govuk::apps::search_admin::override_search_location: NULL
 govuk::apps::whitehall::override_search_location: NULL
 
+govuk::apps::rummager::enabled: false
```

Comment:
Should this be `govuk::apps::rummager::enable_rummager: false`?
https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk/manifests/apps/rummager.pp#L108

Reply:
Yes, it should be.  Damn lack of consistency between app manifests :D.

Reply:
Rummager is old :(